src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
The Circle class should not start with an empty line. Remove the empty line before the class declaration to adhere to the coding standards.